include/cuda/std/detail/libcxx/include/__mdspan/trait_backports.hpp

```diff
@@ -0,0 +1,162 @@
+//
+// 2. Redistributions in binary form must reproduce the above copyright
+// notice, this list of conditions and the following disclaimer in the
+// documentation and/or other materials provided with the distribution.
+//
+// 3. Neither the name of the Corporation nor the names of the
+// contributors may be used to endorse or promote products derived from
+// this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY SANDIA CORPORATION "AS IS" AND ANY
+// EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+// PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL SANDIA CORPORATION OR THE
+// CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+// PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+// LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+// NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+// SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// Questions? Contact Christian R. Trott (crtrott@sandia.gov)
+//
+// ************************************************************************
+//@HEADER
+*/
+
+#ifndef _LIBCUDACXX___MDSPAN_TRAIT_BACKPORTS_HPP
+#define _LIBCUDACXX___MDSPAN_TRAIT_BACKPORTS_HPP
+
+#include "macros.hpp"
+#include "config.hpp"
+
+#ifndef __cuda_std__
+#include <type_traits>
+#include <utility> // integer_sequence
+#endif
+
+//==============================================================================
+// <editor-fold desc="Variable template trait backports (e.g., is_void_v)"> {{{1
+
+#ifdef __MDSPAN_NEEDS_TRAIT_VARIABLE_TEMPLATE_BACKPORTS
+
+#if __MDSPAN_USE_VARIABLE_TEMPLATES
+namespace std {
+
+#define __MDSPAN_BACKPORT_TRAIT(__TRAIT) \
+  template <class... _Args> __MDSPAN_INLINE_VARIABLE constexpr auto __TRAIT##_v = __TRAIT<_Args...>::value;
+
+__MDSPAN_BACKPORT_TRAIT(is_assignable)
+__MDSPAN_BACKPORT_TRAIT(is_constructible)
+__MDSPAN_BACKPORT_TRAIT(is_convertible)
+__MDSPAN_BACKPORT_TRAIT(is_default_constructible)
+__MDSPAN_BACKPORT_TRAIT(is_trivially_destructible)
+__MDSPAN_BACKPORT_TRAIT(is_same)
+__MDSPAN_BACKPORT_TRAIT(is_empty)
+__MDSPAN_BACKPORT_TRAIT(is_void)
+
+#undef __MDSPAN_BACKPORT_TRAIT
+
+} // end namespace std
+
+#endif // __MDSPAN_USE_VARIABLE_TEMPLATES
+
+#endif // __MDSPAN_NEEDS_TRAIT_VARIABLE_TEMPLATE_BACKPORTS
+
+// </editor-fold> end Variable template trait backports (e.g., is_void_v) }}}1
+//==============================================================================
+
+//==============================================================================
+// <editor-fold desc="integer sequence (ugh...)"> {{{1
+
+#if !defined(__MDSPAN_USE_INTEGER_SEQUENCE) || !__MDSPAN_USE_INTEGER_SEQUENCE
+
+namespace std {
```

Comment:
Should be `_LIBCUDACXX_BEGIN_NAMESPACE_STD`

Reply:
This file is removed since we don't need C++11 support.